src/tests/test_dynamic_api/test_views_mvt.py

```diff
@@ -0,0 +1,94 @@
+import mapbox_vector_tile
+import pytest
+from schematools.contrib.django import models
+
+from dso_api.dynamic_api.permissions import fetch_scopes_for_dataset_table, fetch_scopes_for_model
+
+
+@pytest.fixture(autouse=True)
+def clear_caches():
+    yield  # run tests first
+    fetch_scopes_for_dataset_table.cache_clear()
+    fetch_scopes_for_model.cache_clear()
+
+
+@pytest.mark.django_db
+class TestDatasetMVTView:
+    """Prove that the MVT view produces vector tiles."""
+
+    CONTENT_TYPE = "application/vnd.mapbox-vector-tile"
+    URL = "/v1/mvt/afvalwegingen/containers/1/0/0.pbf"
+
+    def test_mvt_index(self, api_client, filled_router, afval_dataset, afval_container):
```

Comment:
```suggestion
    def test_mvt_index(self, api_client, afval_container, filled_router):
```

Dit ga je nodig hebben als mijn test versnelling in master gemerged wordt.